lienzo-tests/src/test/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/WiresLineSpliceControlImplTest.java

```diff
@@ -645,6 +647,66 @@ public void testExecuteSpliceWithContainment() {
 
         // Check containment after
         assertEquals(1, parent.getChildShapes().size(), 0);
+        verify(index, never()).findShapeAt(WiresLineSpliceControlImpl.XY_OFFSET,
+                                           -WiresLineSpliceControlImpl.XY_OFFSET);
+    }
+
+    @Test
+    public void testExecuteSpliceWithContainmentWithDeviation() {
```

Comment:
There are many common lines of code in `testExecuteSpliceWithContainmentWithoutDeviation` and `testExecuteSpliceWithContainmentWithDeviation` test methods. Would it make sense to you to refactor it a little bit? You could create a method for common code for example.